src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Default {
```

Comment:
The class name 'Rectangle' suggests it should have properties related to a rectangle, such as 'length' and 'width', rather than 'side', 'firstLeg', and 'secondLeg'. Consider renaming these variables to better reflect the properties of a rectangle.